examples/gno.land/r/demo/mail/mail.gno

```diff
@@ -0,0 +1,42 @@
+package mail
+
+import (
+	"std"
+	"strconv"
+	"strings"
+	"time"
+
+	"gno.land/p/demo/ufmt"
+)
+
+type Mail struct {
+	id        int
```

Comment:
I'm not sure we should be using an `int` for a Mail ID - considering this can be something that is potentially used a lot as a message passing realm. Message IDs should probably be something derived from the mail message itself, rather than a counter

Reply:
Me neither.
But if `int` in gno means `int64`, and since 2^63 is `9,223,372,036,854,775,808`.
Let's say the system is so popular it generates one million messages per seconds.
It would take 92233720368 seconds to overflow, that is 2924 years.
While it's not perfect (one [source](https://earthweb.com/how-many-emails-are-sent-per-day/) claims more than 3 millions email were sent per seconds in 2023) I think this is a bigger discussion.

- hash(msg): not good IMO (collisions will be trivial to find in the future),
- the transaction number that produced the message may seem good, except when the system can send a message to several recipients, it will need a suffix to work.

As it's WIP and we have staging testnets I think we can keep an int for the time being.

Reply:
Let's  use `int` for now and later switch to `bigint` (#764), maybe.